src/components/ArticlesList/ArticleCard.jsx

```diff
@@ -10,7 +15,11 @@ function ArticleCard({ article_item }) {
         <h3>{article_item.title}</h3>
         <h4>{article_item.topic}</h4>
         <p className="p-text-ellipsis">{article_item.body}</p>
-        <button type="button">Read Article</button>
+        <Link to="/article">
```

Comment:
Make the link here to /article/${article.article_id} - then match that link in your routes (I've suggested above)